README.md

```diff
@@ -1,97 +1,280 @@
-# HTML & CSS3 Course
+<a name="readme-top"></a>
 
-If you are not familiar with linters, read [root level README](../README.md).
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-## Set-up GitHub Actions
+Modify this file to match your project and remove sections that don't apply.
 
-Please do the following **steps in this order**:
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+OPTIONAL SECTIONS:
+- FAQ
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+After you're finished please remove all the comments and instructions!
+-->
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- Just as the previous reviewer highlighted, please remove the default microverse logo after you set up your README.md file. Alternatively, you can replace it with your own logo if you have one :100: Doing this makes your repository more impressive and professional :wink: